tools/hisat/hisat.xml

```diff
@@ -0,0 +1,344 @@
+<tool id="hisat" name="HISAT" version="1.0.0">
+    <description>spliced aligner</description>
+    <macros>
+        <import>hisat_macros.xml</import>
+    </macros>
+    <requirements>
+        <requirement type="package" version="0.1.5">hisat</requirement>
+        <requirement>samtools</requirement>
```

Comment:
Possibly, now that you mention it. I'll update it to depend on either 0.1.18 or 0.1.19 by IUC.